src/FileFetcher.php

```diff
@@ -32,7 +39,13 @@ public function fetch($version, $destination) {
     array_walk($this->filenames, function ($filename) use ($version, $destination) {
       $url = $this->getUri($filename, $version);
       $this->fs->ensureDirectoryExists($destination . '/' . dirname($filename));
+      $this->io->write("Going to download the file $filename");
+      $this->io->write("  from: $url");
+      $this->io->write("  to: $destination/$filename");
```

Comment:
You better change this into a single line
```
$this->io->write("Fetching $url into $destination/$filename");
```
That makes the output shorter in lines and allows to grep for ie .htaccess and get the full line.

Reply:
@clemens-tolboom, thanks for the review, but the comment is not on the last codebase. Those lines had changed in https://github.com/drupal-composer/drupal-scaffold/pull/65/commits/e424668d2360958b52417f72a4d9824b9ff170c6.

Reply:
Hmmm ... guess I'm commenting through the wrong page. (darn) ... let's delete this crap